build/reference/9049DuplicatedMethodName.txt

```diff
@@ -0,0 +1,21 @@
+W049 Duplicated Method Name
+Errors and Warnings
+noreferences
+
+@@description
+
+<h2>Umple semantic warning reported when two methods have the same class names and types</h2>
```

Comment:
I think the word `class` is not needed in the following sentence.
`methods have the same class names`